src/tmplits/tmplits-tabs/README.md

```diff
@@ -0,0 +1,39 @@
+# TmplitTabs
+
+This is a tab control for tmplits. It has selectable tabs and a content area that is shown when the tab is selected.
+
+![example image](./example.png)
+
+## Usage
+
+The lui-tab-control is a web component. The properties are:
+
+* value: The index of the selected tab. Starts at 0.
```

Comment:
Could have a better name? startTabIndex?

Reply:
it's not the start tab though, it's the selected tab, and i used value so that it would behave as an "input". That means you can just databind directly to it with LUX

Reply:
gotcha then maybe selectedTabIndex?